client/components/jetpack-decorative-card/index.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import cloudIcon from 'calypso/assets/images/jetpack/cloud-icon.svg';
+import decorativeCardBackground from 'calypso/assets/images/jetpack/decorative-card-background.jpg';
+import decorativeCardIcon from 'calypso/assets/images/jetpack/decorative-card-icon.svg';
+
+interface Props {
+	iconPath: string;
+}
+
+import './style.scss';
+
+const JetpackDecorativeCard: React.FC< Props > = ( props ) => {
+	const { iconPath } = props;
+
+	return (
+		<div className="jetpack-decorative-card">
+			<div className="jetpack-decorative-card__header">
+				<img
+					alt="" // This is a decorative image
```

Comment:
🤔 I'm not sure that a comment inside a JSX tag [is safe](https://stackoverflow.com/questions/30766441/how-to-use-comments-in-react). I think you have to escape it to avoid it ending up in the HTML.

```suggestion
					alt="" {/* This is a decorative image */}
```

Reply:
Hmmm, doing this:
```
alt="" {/* This is a decorative image */}
```
produces a syntax error for me.

I think escaping comments with `{}` is needed when placing them as a child of a JSX element, but not in the props declaration of an element 🤔 

Would be needed in this case:
```
<div>
    { /* A comment about the text */ }
    <p>Some text</p>
<div>
```

Does that seem correct?

Reply:
Yeah, it's a topic I'm not certain about, but I think your suggestion would work and would feel better to me.

Reply:
Done.